pkg/backend/node/status_node_services.go

```diff
@@ -412,10 +427,16 @@ func (b *StatusNode) personalService() *personal.Service {
 
 func (b *StatusNode) TimeSource() timesource.Provider {
 	if b.timeSourceSrvc == nil {
-		if privateMode {
-			b.timeSourceSrvc = timesource.LocalService()
-		} else {
+		thirdpartyServicesEnabled, err := b.ThirdpartyServicesEnabled()
+		if err != nil {
+			b.logger.Error("failed to get if thirdparty services enabled", zap.Error(err))
+			return timesource.LocalService()
```

Comment:
Returning here won't actually set the `b.timeSourceSrvc`, which we should do
We can just set `thirdpartyServicesEnabled` to false and let the next code do the job
```suggestion
			b.logger.Error("failed to get if thirdparty services enabled", zap.Error(err))
			thirdpartyServicesEnabled = true
```

Reply:
Done!